utils/utils.go

```diff
@@ -178,3 +179,13 @@ func SortSlice[T constraints.Ordered](slice []T) {
 		return slice[i] < slice[j]
 	})
 }
+
+// ConvertTo18Decimals converts a big.Int to 18 decimals from 6
+func ConvertTo18Decimals(amount big.Int) *big.Int {
```

Comment:
nit: In case we should ever support something else, it could be good to be specific

```suggestion
// Convert6To18Decimals converts a big.Int to 18 decimals from 6
func Convert6To18Decimals(amount big.Int) *big.Int {
```